components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -100,6 +100,17 @@ const MyFeed: React.FC<Props> = ({ currentUser }) => {
     greetingLanguage = currentUser.languages[0].language.name
   }
 
+  const resetPagination = () => {
+    // filter out page key to reset the url
+    router.push({...router, query: Object.keys(router.query).reduce((newQuery, key) => {
```

Comment:
```suggestion
    // filter out page key to reset the url
    const newQuery = {...query}
    del newQuery.page
    router.push({...router, query: newQuery })
```

Would work as well, seems a little more direct to me, but either way is good in my book

Reply:
@robin-macpherson + @Lanny -- Thanks for the notes!

I'll go ahead and add in Lanny's suggestion -- new to typescript, so I wouldn't have caught in sublime. I'll also make a point to run `npm run build` in the future -- Just assumed that was done for me by git hooks. Is there travis or some other ci hooked up to this repo?

I'll make a note about `npm run build` in the [contributing-guide.md ](https://github.com/Journaly/journaly/blob/master/docs/contributing-guide.md) and add it to this PR as well.